### ISSUE TITLE
no-done-callback false positive with jest-cucumber

### ISSUE THREAD

**Issue by potb** [jest-community/eslint-plugin-jest#1095]
This code leads to a false-positive since a callback is passed to a function named ``test``.
The warning disappears when the ``test`` function is renamed to something else.

jest: 27.5.1
eslint-plugin-jest: 26.1.4

```ts
import { defineFeature, loadFeature } from "jest-cucumber";

const feature = loadFeature("some/feature");

defineFeature(feature, (test) => {
  test("A scenario", ({ given, when, then }) => {});
});
```

**Comment by G-Rath** [comment]
This is a duplicate of #556 - I've got a draft PR up that should be landed "soon", but I'm the meantime you can either call the method something else or not run eslint-plugin-jest on these files per our readme.

**Comment by potb** [comment]
Alright, thanks.

**Comment by G-Rath** [comment]
@potb this is now resolved :)

**Comment by potb** [comment]
> @potb this is now resolved :)

Thanks for letting me know
